sidebars/pipelines.js

```diff
@@ -56,6 +56,11 @@ const sidebar = [
             type: "doc",
             id: "pipelines/using-pipelines/index",
           },
+          {
+            label: "Multiple Infrastructure-Live Repos",
+            type: "doc",
+            id: "pipelines/multi-account/index",
+          },
```

Comment:
This feels like it goes beyond the scope of "getting started." Should we introduce a new section called "Configuration" and put it in there? Perhaps we could also move the following content to that section:

- The [GitHub Enterprise Users section](https://deploy-preview-1142--pensive-meitner-faaeee.netlify.app/pipelines/using-pipelines/#github-enterprise-users) of the "Using Pipelines" page (that could go under its own page called "GitHub Enterprise")
- Usage data
- Repository access (maybe?)

Reply:
I moved some stuff around mostly into security, let me know if you like this layout better